### ISSUE TITLE
Multiple white spaces in the FSFULLRWD license name

### ISSUE THREAD

**Issue by doki-nordic** [spdx/license-list-XML#1908]
The FSFULLRWD license contains a name with a line break and indention in the middle of the name. According to the standard (https://www.w3.org/TR/REC-xml/#AVNormalize) this is normalized as four spaces:

```
FSF Unlimited License (With License Retention    and Warranty Disclaimer)
```

The source file:
https://github.com/spdx/license-list-XML/blob/62a23e75a55aa08cb2cfdc49ffcc7d5c6c32242f/src/FSFULLRWD.xml#L4-L5

You can see the effect e.g. here:
https://github.com/spdx/license-list-data/blob/main/json/details/FSFULLRWD.json

Also, if you look at source code of the following web page, you will see that too. The HTLM collapses multiple whitespaces into one space, so it is not visible directly in the page:
https://spdx.org/licenses/FSFULLRWD.html

As you can see, this is not an issue for HTML, but it might be for other use cases.

**Comment by swinslow** [comment]
Thank you @doki-nordic! I agree this appears to be a bug, and I'm +1 on fixing this.

Would you like to submit a PR to fix this in https://github.com/spdx/license-list-XML/blob/main/src/FSFULLRWD.xml?

**Comment by doki-nordic** [comment]
> Would you like to submit a PR to fix this in https://github.com/spdx/license-list-XML/blob/main/src/FSFULLRWD.xml?

No problem, the PR: https://github.com/spdx/license-list-XML/pull/1910

**Comment by swinslow** [comment]
LGTM, thank you @doki-nordic!

**Comment by swinslow** [comment]
Fixed in #1910
